### DISCUSSION TITLE
Additional Signals on the Indicators

### DISCUSSION THREAD

**Discussion by RobertWeaver** [DaveSkender/Stock.Indicators#822] [Ideas]
[StrategyQuant](https://strategyquant.com/), [AlgoWizard](https://algowizard.io/) and [VectorBt](https://github.com/polakowo/vectorbt#sparkles-usage) have these signals on the  indicators like below...

Is this something we could add to the output result from the Indicator? 😅 

From [StrategyQuant](https://strategyquant.com/)
![image](https://user-images.githubusercontent.com/6476560/173474354-16b3cd84-078b-4666-ad77-736e63375390.png)

From [AlgoWizard](https://algowizard.io/) 
![image](https://user-images.githubusercontent.com/6476560/173474983-698126e7-91d8-4664-9370-d84470f438aa.png)

From [VectorBt](https://github.com/polakowo/vectorbt#sparkles-usage) 
```python
fast_ma = vbt.MA.run(price, 10)
slow_ma = vbt.MA.run(price, 50)
entries = fast_ma.ma_crossed_above(slow_ma)
exits = fast_ma.ma_crossed_below(slow_ma)

pf = vbt.Portfolio.from_signals(price, entries, exits, init_cash=100)
pf.total_profit()
```

From [VectorBt Pro](https://vectorbt.pro/features/#merging)
```python
def strategy2(data):
     bbands = vbt.BBANDS.run(data.get("Close"), window=14)
     entries = bbands.close_crossed_below(bbands.lower)
     exits = bbands.close_crossed_above(bbands.upper)
     return vbt.Portfolio.from_signals(
         data.get("Close"), 
         entries, 
         exits, 
         init_cash=200
     )
```

**Comment by DaveSkender** [comment]
Yes.  This is something I plan to take on in stages.  There’s several components to a trading system, as you can see in your examples, with the indicator as the basic prerequisite for the kind of analysis you’re seeking.  This library is just one building block and we’re already [thinking about further intrinsic analysis](https://github.com/DaveSkender/Stock.Indicators/discussions/320#discussioncomment-2749534), a separate library for signals, and a [backtest engine](https://github.com/DaveSkender/Stock.Indicators/discussions/484) as the next components to build.

One thing at a time!  I still have some major features to do in [this library’s roadmap](https://github.com/DaveSkender/Stock.Indicators/discussions/218) before moving on.  With that said, we’re always seeking additional contributors!
